DigitalOffice/Tests/UserServiceTests/Commands/GetUserByEmailCommandTests.cs

```diff
@@ -0,0 +1,97 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.UserService.Commands;
+using LT.DigitalOffice.UserService.Commands.Interfaces;
+using LT.DigitalOffice.UserService.Database.Entities;
+using LT.DigitalOffice.UserService.Mappers.Interfaces;
+using LT.DigitalOffice.UserService.Models;
+using LT.DigitalOffice.UserService.Repositories.Interfaces;
+using LT.DigitalOffice.UserService.Validators;
```

Comment:
Using directive is not required by the code and can be safely removed

Reply:
Fixed